packages/client-sdk-nodejs/src/preview-leaderboard-client.ts

```diff
@@ -30,12 +31,22 @@ export class PreviewLeaderboardClient implements ILeaderboardClient {
       ...props,
       configuration: configuration,
     };
+    this.configuration = configuration;
 
     this.logger = configuration.getLoggerFactory().getLogger(this);
     this.logger.debug('Creating Momento LeaderboardClient');
     this.dataClient = new LeaderboardDataClient(propsWithConfig, '0'); // only creating one leaderboard client
   }
 
+  public close() {
+    this.dataClient.close();
+    this.configuration.getMiddlewares().forEach(m => {
+      if (m.close) {
+        m.close();
+      }
+    });
+  }
+
   /**
    * Creates an instance of LeaderboardClient with 32-bit float scores.
```

Comment:
lol could you correct this comment while you're in here? should be 64-bit float scores now

> Creates an instance of LeaderboardClient with 64-bit float scores.